test/src/sdk-harness/test_projects/storage_vec/b256/mod.rs

```diff
@@ -0,0 +1,263 @@
+mod utils;
+
+use utils::{
+    setup::get_contract_instance,
+    wrappers::{clear, get, insert, is_empty, len, pop, push, remove, swap_remove},
+};
+
+// TODO: Replace many of the get calls with direct storage values
+// once the SDK supports that
+
+const BYTES1: [u8; 32] = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31];
```

Comment:
Is there any particular reason for these numbers being chosen? They seem to be incremented by 1 so I assume not. I suggest populating the entire array with the same value like

```rust
const BYTES1: [u8; 32] = [1; 32];
const BYTES2: [u8; 32] = [2; 32];
const BYTES3: [u8; 32] = [3; 32];
const BYTES4: [u8; 32] = [4; 32];
const BYTES5: [u8; 32] = [5; 32];
```